### DISCUSSION TITLE
Can' access images in public folder

### DISCUSSION THREAD

**Discussion by NyiNyiAung16** [nuxt/nuxt#29635] [Questions]
My issue is that can't access images in public folder after creating a project in preview building. I work totally fine in development. I already have images in public folder. After building the project, I can access that images. But creating a project api and store the images in public folder. The images store totally fine but can't access in web.

**Comment by xjccc** [comment]
[public](https://nuxt.com/docs/guide/directory-structure/public)

The newly created resource cannot be accessed, only existing resources can be accessed.

**Reply by NyiNyiAung16** [reply]
yes so how can i fix the issue

**Reply by xjccc** [reply]
You can't accessed by this way. I think you could upload to OSS and accessed it.
